crs_application/src/task_managers/process_execution_manager.cpp

```diff
@@ -232,7 +232,7 @@ common::ActionResult ProcessExecutionManager::execMediaChange()
     run_robot_script_req->filename = config_->ur_tool_change_script;
     std::shared_future<crs_msgs::srv::RunRobotScript::Response::SharedPtr> run_robot_script_future =
         run_robot_script_client_->async_send_request(run_robot_script_req);
-    std::future_status run_robot_script_status = run_robot_script_future.wait_for(std::chrono::seconds(30));
+    std::future_status run_robot_script_status = run_robot_script_future.wait_for(std::chrono::seconds(600));
```

Comment:
Can you make the 600 int a global constant so that it is easier to locate?

Reply:
Fixed